test/servico/cst01Test.java

```diff
@@ -52,23 +51,25 @@ public void tearDown() {
     }
     
     @Test
-    public void testeNotificaçãoDeAusência_ÚnicoHorário() throws ParseException
-    {
+    public void testeNotificaçãoDeAusência_ÚnicoHorário() throws ParseException{
+        
+        //Login com usuário: Funcionario
         LoginService loginService = new LoginService();
-        assertTrue(loginService.VerificarUsuarioESenha("Funcionario1", "123456"));
+        assertTrue(loginService.VerificarUsuarioESenha("Funcionario", "123456"));
         
         ProfessorService professorService = new ProfessorService();
         ProfessorModel professor = professorService.obterProfessorPorNome("Professor1");
         
-        NotificacaoService notificaçãoService = new NotificacaoService();
+        //Funcionalidade de "Notificação de Ausência" 
+        NotificacaoService notificaçãoService = new NotificacaoService(); 
         List<AusenciaModel> ausencias = notificaçãoService.listarAusencias();        
         assertEquals(0, ausencias.size());
         
-        notificaçãoService.notificarAusencia(professor.id, "25/11/2013 20:01", "25/11/2013 21:29", "Motivo Declarado", new LinkedList<String>());
+        notificaçãoService.notificarAusencia(professor.id, "13/06/2016 20:05", "13/06/2016 21:30", "Motivo: Palestra", new LinkedList<String>());

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 120 characters (found 145).](https://www.codacy.com/app/calebepb/substituicao/file/2779914335/issues/source?bid=3364462&fileBranchId=3364481#l68)